healthcheck.js

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env node
+'use strict'
+require('dotenv').config()
+
+const http = require('http')
+
+const signale = require('./src/utils/signale')
+const checkMongoDB = require('./src/utils/connect')
+
+const port = process.env.ACKEE_PORT || process.env.PORT || 3000
+const dbUrl = process.env.ACKEE_MONGODB || process.env.MONGODB_URI
+const serverUrl = `http://localhost:${ port }`
+
+if (dbUrl == null) {
+	signale.fatal('MongoDB connection URI missing in environment')
+	process.exit(1)
+}
+
+const checkHTTP = (url) => {
+	return new Promise((resolve, reject) => {
+		const req = http.request(url, (res) => {
+			if (res.statusCode < 200 || res.statusCode >= 300) {
+				return reject(new Error(`Status Code: ${ res.statusCode }`))
+			} else {
+				return resolve(`Conntected to ${ url }`)
+			}
+		})
+		req.on('error', reject)
+		req.end()
+	})
+}
+
+const checkAPI = (url) => {
+	return new Promise((resolve, reject) => {
+		const req = http.request(url, (res) => {
```

Comment:
Ackee contains the `node-fetch` package. We could use it here, too.

Reply:
+ async/await instead of `new Promise` to keep it in sync with the rest of Ackee.

Reply:
Ah, I didn't notice. I'll replace it